<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TX717-4441WGD" name="testscripts.SF.RegressionTestScriptsOpp.TC_SF_VASEMEABrokerAutoGeneratedOpportunity" tests="1" failures="0" timestamp="7 Sep 2017 17:21:45 GMT" time="1.009" errors="1">
  <testcase name="runTC_SF_VASEMEABrokerAutoGeneratedOpportunity" time="1.009" classname="testscripts.SF.RegressionTestScriptsOpp.TC_SF_VASEMEABrokerAutoGeneratedOpportunity">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.cognizant.Craft.DriverScript.quitWebDriver(DriverScript.java:845)
	at com.cognizant.Craft.DriverScript.driveTestExecution(DriverScript.java:157)
	at testscripts.SF.RegressionTestScriptsOpp.TC_SF_VASEMEABrokerAutoGeneratedOpportunity.runTC_SF_VASEMEABrokerAutoGeneratedOpportunity(TC_SF_VASEMEABrokerAutoGeneratedOpportunity.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- runTC_SF_VASEMEABrokerAutoGeneratedOpportunity -->
</testsuite> <!-- testscripts.SF.RegressionTestScriptsOpp.TC_SF_VASEMEABrokerAutoGeneratedOpportunity -->
